src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -198,9 +230,40 @@ object TelemetryTypes extends Enumeration {
   val SPARK_STREAMING_END: Value = Value("spark_streaming_end")
   val SPARK_EGRESS: Value = Value("spark_egress")
   val SPARK_CLIENT_INFO: Value = Value("spark_client_info")
+  val SPARK_QUERY_STATUS: Value = Value("spark_query_status")
   val SPARK_PUSHDOWN_FAIL: Value = Value("spark_pushdown_fail")
 }
 
+object TelemetryConstValues {
+  val OPERATION_READ = "read"
+  val OPERATION_WRITE = "write"
+  val STATUS_SUCCESS = "success"
+  val STATUS_FAIL = "fail"
+}
+
+object TelemetryQueryStatusFields {
+  // Spark connector version
+  val SPARK_CONNECTOR_VERSION: String = "spark_connector_version"
+  // The query operation: read vs write
+  val OPERATION: String = "operation"
+  // Query statement
+  val QUERY: String = "query"
+  // query ID if available
+  val QUERY_ID: String = "queryid"
```

Comment:
query_id. Please have consistent style

Reply:
I will use QUERYID to align OOB message.